### PR TITLE
Seismic Configuration (Verified Publisher)

### PR DESCRIPTION
**Pull request by rkushvaha**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by rkushvaha** [comment]
@microsoft-github-policy-service agree [company="Seismic"]

**Comment by rkushvaha** [comment]
@microsoft-github-policy-service agree company="Seismic"

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+    "application/json"
+  ],
+  "paths": {
+    "/contentProperties/{contentPropertyId}/values": {
+      "get": {
+        "tags": [
+          "Content Properties"
+        ],
+        "summary": "Content properties - Get values of a content property",
+        "description": "Provides the list of content property values for a given content property. This will only return values for properties that have a defined list of values. Properties which have no defined domain of values and are filled in freeform by users are not provided here.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json:34]: @rkushvaha we highly suggest you keep the all operations' descriptions short to fit in the box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

File: certified-connectors/Seismic Configuration/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "0718e920-53ef-48e9-994c-504906a6e540",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Seismic Configuration/apiProperties.json:8]: @rkushvaha secret look alike values are not allowed. please replace with dummy values.

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,807 @@
...
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Content Properties"
+        ],
+        "summary": "Content properties - Add values to a content property",
+        "description": "Add one or more content property values to an existing content property. Property values cannot be added to properties where an existing domain of values does not exist. Property values are case sensitive.\n#### Warning:\nAdding a domain of values to an existing property that does not have an domain of values can result in a loss of data. If this is desired please log into Seismic and create a domain of values to enable this functionality.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Seismic Configuration/apiDefinition.swagger.json:66]: @rkushvaha all files should be free of grammatical and spelling errors. in the description you have "...that does not have an domain of values..." please change to "...that does not have **a domain** of values...".

File: certified-connectors/Seismic Configuration/readme.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## Prerequisites
+You must have user created in Seismic tenant.
+
+## Supported Operations
+### Get Content Property Values
+Provides the list of content property values for a given content property. This will only return values for properties that have a defined list of values. Properties which have no defined domain of values and are filled in freeform by users are not provided here.
+
+### Add Content Property Values
+Add one or more content property values to an existing content property. Property values cannot be added to properties where an existing domain of values does not exist. Property values are case sensitive.#### Warning:Adding a domain of values to an existing property that does not have an domain of values can result in a loss of data. If this is desired please log into Seismic and create a domain of values to enable this functionality.
```
Comment (Amjed-Ayoub) [inline certified-connectors/Seismic Configuration/readme.md:14]: @rkushvaha all files should be free of grammatical and spelling errors. in the description you have "...that does not have an domain of values..." please change to "...that does not have **a domain** of values...".

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json (hunk 1 above)
Comment (rkushvaha) [reply certified-connectors/Seismic Configuration/apiDefinition.swagger.json]: @Amjed-Ayoub description updated.

File: certified-connectors/Seismic Configuration/apiDefinition.swagger.json (hunk 3 above)
Comment (rkushvaha) [reply certified-connectors/Seismic Configuration/apiDefinition.swagger.json]: @Amjed-Ayoub long description removed,

File: certified-connectors/Seismic Configuration/apiProperties.json (hunk 2 above)
Comment (rkushvaha) [reply certified-connectors/Seismic Configuration/apiProperties.json]: Added dummy value

**Comment by rkushvaha** [comment]
Hi @Amjed-Ayoub all review comments are resolved. Can you please help to review again, Thanks.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @rkushvaha,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**